### DISCUSSION TITLE
Is there a way to switch between alternate <START>s?

### DISCUSSION THREAD

**Discussion by brknsoul** [LostRuins/koboldcpp#1088] [Q&A] [answered]
Some character cards have multiple \<START\> blocks. Without using ST, et al, does koboldcpp provide a way to switch between these?

![image](https://github.com/user-attachments/assets/88083d17-e028-44d7-a905-c7f944ad4686)

**Comment by LostRuins** [comment, marked as answer]
I assume these are set as alternate greetings? For v2 cards, you're now able to select your own custom greeting message if multiple are present. Simply load the card and it should allow you to pick. 

![image](https://github.com/user-attachments/assets/e988ed11-4107-4a6c-b5e2-455c252afcb0)

**Reply by brknsoul** [reply]
OK, that works. It'd be nice if the Chat/Instruct mode selector was detached (ie: separate option), and Greeting Message UI was better.

The example you've given, options 1 through 9 seem the same, but I imagine they differ further along in the greeting.

Perhaps a dropdown box, that shows the short greeting in the box, then the full greeting underneath, rather than typing a number.

**Reply by LostRuins** [reply]
this specific card is just a bad example. generally, the messages are sufficiently different that picking isn't an issue.

**Reply by brknsoul** [reply]
OK, I've marked an answer and submitted a Feature Request. Thank you!
